lib/intercom-rails/script_tag.rb

```diff
@@ -64,7 +64,7 @@ def intercom_settings
       hsh[:session_duration] = @session_duration if @session_duration.present?
       hsh[:widget] = widget_options if widget_options.present?
       hsh[:company] = company_details if company_details.present?
-      hsh[:hide_default_launcher] = Config.hide_default_launcher if Config.hide_default_launcher
+      hsh[:hide_default_launcher] = Config.hide_default_launcher.call(user_details) if Config.hide_default_launcher.call(user_details)
```

Comment:
Couldn't this have been just `hsh[:hide_default_launcher] = Config.hide_default_launcher.call(user_details)` to avoid calling the proc twice on a single request?